#defines floating point for the position variables
float32 pos_x
float32 pos_y
